Backend/controllers/productservices.js

```diff
@@ -7,14 +7,14 @@ import productService from "../models/productservices";
  */
 
 export const createProductService = async (req, res) => {
-    const productServiceData = req.body;
-    const createProductService = new productService(productServiceData);
-    try {
-        await createProductService.save();
-        req.status(201).json(new productService);
-    } catch (error) {
-        res.status(409).json({ message: error.message });
-    }
+  const productServiceData = req.body;
```

Comment:
I think if you just add some error handling here if there is no middleware for that and some input validation and check as well